src/searcher/censys.ts

```diff
@@ -18,22 +18,29 @@ export class Censys implements Searcher {
   public searchByASN(query: string): string {
     const asn = extractASNumber(query);
     return buildURL(this.baseURL, "/search", {
-      q: `autonomous_system.asn:${asn}`,
+      q: `autonomous_system.asn: ${asn}`,
       resource: "hosts",
     });
   }
 
   public searchByDomain(query: string): string {
     return buildURL(this.baseURL, "/search", {
-      q: `parsed.names:${query}`,
+      q: `names: ${query}`,
       resource: "certificates",
     });
   }
 
   public searchByEmail(query: string): string {
     return buildURL(this.baseURL, "/search", {
-      q: `parsed.subject.email_address:${query}`,
+      q: `parsed.subject.email_address: ${query}`,
       resource: "certificates",
     });
   }
+
+  public searchByGATrackID(query: string): string {
+    return buildURL(this.baseURL, "/search", {
+      q: `services.http.response.body: "${query}"`,
+      resource: "hosts",
+    });
+  }
```

Comment:
I understand your intent. But I think it's not good. It's misleading for users.
I think many users think that Censys has a special feature to search hosts by GA track ID, but it doesn't. 
Also, if I allow this, I should add the same method for Shodan, Onyphe, etc. It doesn't make much sense to me.

So could you remove this function and update docs, please?

Reply:
Totally fair, I will update it.